README.md

```diff
@@ -1,10 +1,58 @@
 ![Nodezoo](https://raw.githubusercontent.com/rjrodger/nodezoo-web/to-redux/client/assets/img/logo-nodezoo.png)
 
 # nodezoo-info
-nodezoo.com micro-service handling module. 
+nodezoo.com micro-service handling module.
+One of the micro-services that implement
+[nodezoo.com](http://nodezoo.com), a search engine for
+[Node.js](http://nodejs.org) modules. The NodeZoo search engine is an
+example of a real-world service built using Node.js
+micro-services. Each micro-service is published in its own github
+repository. The codebase is intended to be used as a larger-scale
+example, and as a starting point for your own projects.
+
+The search-engine is under development in an open manner, and a blog
+series (see below) covers the full details.
+
+This micro-service uses the Seneca micro-service toolkit. For an
+introduction to Seneca itself, see the
+[senecajs.org](http://senecajs.org) site.
+
+Feel free to contact me on twitter if you
+have any questions! :) [@rjrodger](http://twitter.com/rjrodger)
+
+Tested on: Seneca 0.5.20, Node 0.10.31
+
 
 - __Sponsor:__ [nearForm][]
 
+## Install
+
+Create a project folder for all the _nodezoo_ micro-service repositories. The _clone_ this repository into your project folder:
+
+```sh
+git clone https://github.com/rjrodger/nodezoo-info.git
+```
+
+Alternatively, fork the repository so that you can make your own changes.
+
+
+## Messages
+
+This micro-service recognizes the following messages:
+
```

Comment:
These are the messages for npm, you need to figure out the ones for this service and put them here.